doc/classes/Node.xml

```diff
@@ -556,6 +556,12 @@
 				[method request_ready] resets it back to [code]false[/code].
 			</description>
 		</method>
+		<method name="is_part_of_edited_scene" qualifiers="const">
```

Comment:
Why `is_part_of_edited_scene` and not `is_inside_edited_scene`? I mean, I kind of get why, but it does not quite roll off the tongue.

Reply:
idk, IMO this name is fine.
Also the method was changed a bit to include siblings of the scene, so `is_inside_edited_scene()` would be less accurate.